python/tvm/relay/frontend/onnx.py

```diff
@@ -5906,7 +5906,16 @@ def from_onnx(
     graph = model.graph
 
     try:
-        opset_in_model = model.opset_import[0].version if model.opset_import else 1
+        opset_in_model = 1
+        if model.opset_import:
+            # TODO: for now we only really support ai.onnx op set
+            # TODO: handle other namespaces well see https://github.com/apache/tvm/issues/10950
+            for opset_identifier in model.opset_import:
+                # As per https://github.com/onnx/onnx/blob/main/docs/IR.md
+                # All operator sets except the default one must specify the operator version
+                if str(opset_identifier.domain) in ["ai.onnx", ""]:
```

Comment:
I think this will still run into some issues. When there is only one `opset_import`, it doesnt have a domain name, just a version. We'll need to add a special case when the length of `model.opset_import` is 1 and just pull out the version directly.

Reply:
Do you mean opset_identifier won't have the attribute "domain", or that it will return the empty string?

Reply:
It could be an earlier ONNX spec which we do not support.

Reply:
It returns an empty string despite having a version that needs to be respected. If we dont handle this case we'll fail the `import_model` test.

Reply:
Hmm in the code, it should be fine with the "", I'll investigate this test.

Reply:
The test fail was due to known error https://github.com/apache/tvm/pull/13067.